projects/storefrontlib/src/lib/checkout/guards/index.ts

```diff
@@ -1,3 +1,7 @@
 import { OrderConfirmationPageGuard } from './order-confirmation-page.guard';
+import { SetShippingAddressGuard } from './set-shipping-address.guard';
```

Comment:
Could you rename it to `shipping-address-set.guard`? Starting with `set` gives the impression that is sets shipping address instead of checking if shipping address is set.

Reply:
rightly